Learn page: what is decentralization? #3025 (#3050)
ethereum/ethereum-org-website: src/content/decentralization/index.md

### Storage (desto, maybe?) {#storage}

Blockchains like Etherereum's aren't the most efficient places for storing data. Luckily, there are various open protocols/paid services that provide decentralized, verifiable locations for your stuff:

- [IPFS](https://ipfs.io/)
- [Filecoin](https://filecoin.io/) (based in part on IPFS)
- [Storj](https://storj.io/)
- [Swarm](https://swarm.ethereum.org/) (also for encrypted messaging)

### File sharing {#filesharing}

- [BitTorrent](https://www.bittorrent.org/introduction.html). No, it's not just for music pirates.

### Other blockchains {#blockchains}

There are other blockchain-based projects out there doing different things (or the same things, but with different marketing approaches). If decentralization is a priority for you, please remember that "blockchain" does not equal "decentralized". A blockchain with a central point of failure (i.e. a single governing authority that can make unilateral decisions) would have many of the same risks as a centralized system.

### Other decentralized projects {#decentralized-projects}

- [Scuttlebot](https://scuttlebot.io/): Logging
- [YaCy](https://yacy.net): Search

> - [Ethereum Naming Service](https://ens.domains/): Name registration

Author: Hey, didn't ignore this — I just really like the idea of these sections becoming places for people to start when building a completely decentralized web project/practice. Do you guys have any suggestions for this? I also added IPNS for some balance here.

> - [Ethereum Naming Service](https://ens.domains/): Name registration

Reviewer: If we're addressing users who may want to build a dapp, we should direct them to our developer docs.

This level of resilience and resistance to attack is possible because of decentralization. Ethereum keeps running (and so do its smart contracts) even if there is a massive cloud provider outage, widespread censorship, or something else that breaks the popular user interface.

## Pros and cons of using Ethereum {#pros-cons}

Ethereum's principles of decentralization result in a lot of freedom. However, there's a tradeoff: usually increased complexity. Nothing's perfect, right?

You can overcome most of the downsides by learning some new behavior as a user. Unfortunately, we all know that learning isn't all that easy. Especially when real money is on the line. You can do it, though!

The good news is that there's a major push to make the decentralized web more user-friendly — some of these "cons" might disappear sooner rather than later. More precisely, some of the complexity will start to happen in computer programs instead of happening right up in your face.

> 

Author: Hey thanks! and I agree this one needs some work. I rewrote this section a little to hit a little harder and focus on empowerment.

- reworked the intro
- expanded the security subsection
- added a section about contributing to projects
- reiterated the trade-off between freedom/control and convenience (and underlined that I think getting more freedom for less convenience is a good trade)

Also I think internet history is a great topic to include in general. I added a section for it with a little soapbox moment down at the bottom.

---
title: What is a decentralized internet?
description: An intro to decentralization on the web
lang: en
emoji: ":world_map:"
sidebar: true
sidebarDepth: 2
summaryPoints:
  [
    "Decentralization is a design principle.",
    "Ethereum is built on, benefits from, and pursues decentralization.",
    "Decentralized and centralized systems can and do coexist.",
  ]
---

## What's decentralization? {#what-is-decentralization}

> Decentralization is a way to create systems in which no individual or group can unilaterally make major decisions, especially not when it hurts the other participants. You can use this principle to build things in the fields of finance, architecture, art, communications, government, and — you guessed it — computer science.

Reviewer 2: My 2 cents - I'd argue decentralization isn't a "way to create systems" - it's an _aspect_ of systems. Every system exists on a spectrum of decentralization. It's a question of to what degree any one individual or group has control over it.

> In basic terms: Nobody is in charge, but things still work. Rules without rulers.

Reviewer 2: ```suggestion
In basic terms: nobody is in charge, but things still work. Rules without rulers.
```

Author: This reminds me — which style guide does ethereum.org use? Probably should have asked before writing 😆

> For example, no one person controls the Ethereum network. It works based on the rules set out in the code/protocol which no single person has the authority to change. In contrast, Amazon Web Services could decide to stop providing server space to any of its clients upon suspected violation of any of a number of terms in its draconian and mutable user-agreement contract.

Reviewer 2: ```suggestion
No one person controls the Ethereum network. It operates based on the rules set out in the protocol which no single person or organization has the authority to change. In contrast, Amazon Web Services could decide to stop providing server space to any of its clients upon suspected violation of any of a number of terms in its draconian and mutable user-agreement contract.
```

Reviewer 2: Is "draconian" a bit harsh?

If you use a smart contract or transact on Ethereum, you're doing something to support the entire network. If you give feedback to a developer, asking for new features or reporting issues, that's even better. You can even contribute directly by documenting, translating, coding, or starting your own project.

### Basic Ethereum safety {#safety-ethereum}

Non-technical? Don't worry. The decentralized financial systems of the future will probably be easier to use than centralized finance is today. In the meantime, please take some basic steps to keep your funds safe when you're starting out on Ethereum:

- [Learn about how wallets work](https://ethereum.org/en/wallets/)
- Confirm (before interacting with a smart contract) that your wallet or exchange account can interact with smart contracts
- Triple-check destination addresses, even if you're sending things to yourself or to exchanges
- Send small test transactions before sending big amounts
- Understand the difference between pseudonymous and private (most blockchains are public, pseudonymous records)

> - Above all else, do not share private keys or mnemonic phrases with others. Not even if they're trying to help. 

Reviewer 3: This could use some additional emphasis (if not it's own page eventually) in my opinion. Also, can imagine not everyone knows what a "mnemonic" is, but will understand something like "12 or 24-words"..
One thought:
> Above all else, **backup and do not share private keys or mnemonic phrases (12 or 24-words) with others.** Not even if they're trying to help. They are the master key to your funds and are for your eyes only. These should **only** ever be typed **directly into your wallet**, nowhere else.
```
Above all else, **backup and do not share private keys or mnemonic phrases (12 or 24-words) with others.** Not even if they're trying to help. They are the master key to your funds and are for your eyes only. These should **only** ever be typed **directly into your wallet**, nowhere else.
```

- **Better communication between network operators, users, and software developers**: Communication helps identify potential uses and resolve issues. Sometimes one group will have a perspective that the others did not consider.

## Not all crypto and blockchain projects are decentralized {#centralized-vs-decentralized}

"Blockchain/crypto" does not equal "decentralized". The fact is that most systems have aspects of centralization and decentralization. These design concepts can and do coexist.

So the question becomes what types of decentralization are practical? What types of centralization do we absolutely need? As mentioned above, Ethereum needs to have a central set of computing rules, or else the machine wouldn't know what to do.

Let's look at a couple of examples.

### Example 1: United States Dollar coins {#usd-on-ethereum}

For example, USDC is a United-States-Dollar-value cryptocurrency. It is "backed by fully reserved assets" (a phrase assumedly meant to communicate that it is fully backed by reserve assets). It is controlled in a relatively centralized way: via a permissioned consortium founded by two big corporations.

> DAI is another dollar-value coin. DAI gets created when people deposit crypto assets in a program (a smart contract). That smart contract was developed by a group called [MakerDAO](https://makerdao.com/en/). People who hold the MKR asset get to vote on how the program behaves. It is far more decentralized than USDC.

Reviewer 3: May be interesting to make a note about recent changes allowing USDC as collateral to take out DAI loans, which arguably shifts the needle of DAI towards being more centralized. Could help to demonstrate that decentralization is not binary, but a spectrum, and different specific areas of the network can operate on different areas of that spectrum, with trade offs along the way.

## Decentralization in Ethereum {#ethereum-decentralized}

First things first: Nobody owns Ethereum. The [Ethereum Foundation](https://ethereum.foundation) provides support, but so do tons of other individuals and organizations.

Building on that, Ethereum is decentralized in many other important ways. People operate nodes across the world, there are many different client software options, and it interoperates (often via third-party [decentralized apps](https://ethereum.org/en/dapps/)) with a lot of other decentralized protocols. It is built to resist spammers, to provide clear consensus, and to be unprofitable (or at least proportionately expensive) to attack. 

Ethereum is also *centralized* in a very important way — there is an agreed-upon set of rules (in the form of computer code) that every node should follow. This allows the system to operate predictably. However, it bears mentioning that these rules are upgraded and maintained in a decentralized way.

> ### Smart contracts {#smart-contracts}

Reviewer: Describing smart contracts in this way at this point in the page feels out of place. What's the purpose behind bringing them up here?

Author: Yeah.... this section did not get my point across, thanks for pointing that out. I expanded on what I was trying to say in the most recent commit.

Behind every dapp (decentralized app) on Ethereum, there is a smart contract. This is a program that runs on the Ethereum network for users to interact with.

Smart contracts are an essential part of Ethereum's decentralized infrastructure. They can theoretically provide any service that a computer can. They can execute currency exchanges, retrieve and display information, make calculations, and so on.

We say that Ethereum can solve "theoretically" any problem because there are some limits: namely the expensive digital real estate in the [Ethereum Virtual Machine](https://ethereum.org/en/glossary/#evm) and on the [blockchain](https://ethereum.org/en/glossary/#blockchain). 

> ## Pros and cons of using Ethereum {#pros-cons}

Reviewer: I think this needs to directly reference Ethereum as a decentralized platform...

Ethereum's principles give incredible importance to decentralization. This has some amazing pros but also some cons, or behaviour that you probably won't be used to as a user of the internet today...

- Ethereum does not associate your activities with your personal identity (because it doesn't know who you are).
- Brute-force "identity theft" crimes can be extremely difficult because account credentials are cryptographically protected.
- You can issue transactions instantly from anywhere with an internet connection.

### Cons {#cons-of-ethereum}

- Nobody but the recipient can help if you send your funds to the wrong address.
- You are responsible for account security and privacy practices: for keeping your [private keys](https://ethereum.org/en/glossary/#private-key) secure and your personal information separate from your online activities.
- You will probably have to learn new information and behaviors. Currently, even the most user-friendly apps can be technically challenging.
- It is your responsibility to identify and comply with applicable laws.
- It can be difficult to know which contracts you should trust.

In the end, it's up to you to decide. However, know that there are a lot of smart people who are dedicating themselves to making this technology easier for you to use. If it's not the best choice for you now, check it out again in a few months.

> ## Trust and consequences {#trust-in-decentralization}

Reviewer: This section feels like it belongs above when we talk about why decentralized services are important

- **Better geographical distribution of network operators (active participants in many different countries)**: Having nodes in many different places reduces the risk of government interference, outages from regional disasters, etc.
- **Distribution of ETH balances (already one of the most even distributions in crypto)**: Currency value is at risk for manipulation when the supply is largely controlled by a single entity or group of associated entities. In other words, a centralized group can decide how much is available. Increased distribution reduces this risk.
- **Increased cooperation between the Ethereum community and other decentralized projects**: Ethereum is at the infrastructure level for the decentralized web. Linking other projects and building more applications increases competition, but it also increases utility.
- **Better communication between network operators, users, and software developers**: Communication helps identify potential uses and resolve issues. Sometimes one group will have a perspective that the others did not consider.

> ## Centralized vs. decentralized {#centralized-vs-decentralized}

Reviewer: This title should better reflect the content, you're talking about how not all blockchain/crypto projects are decentralized.
